src/rbr/Region.java

```diff
@@ -109,18 +88,7 @@ private static String getIpMerged(Region r1, Region r2) {
 	}
 
 	boolean canBeMergedWith(Region that) {
-		return (this.box().isContiguous(that.box()) && opIsSubset(this, that));
-	}
-
-	private static boolean opIsSubset(Region r1, Region r2) {
-		String r1Op = sortStrAlf(r1.outputPorts());
-		String r2Op = sortStrAlf(r2.outputPorts());
-		return (r1Op.contains(r2Op) || r2Op.contains(r1Op));
-	}
-
-	private static String sortStrAlf(String input) {
-		char[] ip1 = input.toCharArray();
-		Arrays.sort(ip1);
-		return String.valueOf(ip1);
+		return (this.box().isContiguous(that.box()) && (this.outputPorts().containsAll(that.outputPorts())
+				|| that.outputPorts().containsAll(this.outputPorts())));

```

Comment:
I'd rather keep the opIsSubset method with these two conditions. Maybe changing its name.